gpu_ocean/SWESimulators/CDKLM16.py

```diff
@@ -125,6 +133,10 @@ def __init__(self, \
             nx = nx + boundary_conditions.spongeCells[1] + boundary_conditions.spongeCells[3] - 2*ghost_cells_x
             ny = ny + boundary_conditions.spongeCells[0] + boundary_conditions.spongeCells[2] - 2*ghost_cells_y
             y_zero_reference_cell = boundary_conditions.spongeCells[2] + y_zero_reference_cell
+            
+        #Compensate f for reference cell
+        f = f - coriolis_beta * y_zero_reference_cell * dy
```

Comment:
This will not work properly if angle is nonzero. I personally don't like y_zero_reference cell, and think that it is up to the user to externally set f and beta to reflect the coriolis parameters of the first cell, eta[0, 0]